apps/examples/security_test/seclink/sl_test_main.c

```diff
@@ -36,9 +36,9 @@ int main(int argc, FAR char *argv[])
 int sl_test_main(int argc, char *argv[])
 #endif
 {
-	sl_keymgr_test();
-	sl_auth_test();
-	sl_ss_test();
+	/* sl_keymgr_test(); */
```

Comment:
You can erase them if these tests are unnecessary anymore.
If not, please add `#ifdef CONFIG_XXXX` to run them for specific cases instead of comment out.